build_node/builders/base_rpm_builder.py

```diff
@@ -262,8 +263,13 @@ def build_packages(self, src_dir, spec_file=None):
             if srpm_build_result:
                 self.save_build_artifacts(srpm_build_result,
                                           srpm_artifacts=True)
-        srpm_path = srpm_build_result.srpm
-        self.logger.info('src-RPM %s was successfully built', srpm_path)
+        if srpm_build_result:
+            srpm_path = srpm_build_result.srpm
+            self.logger.info('src-RPM %s was successfully built', srpm_path)
+        elif self.task.built_srpm_url:
+            srpm_path = built_srpm_url
+            self.logger.info('src-RPM %s is already built', built_srpm_url)
+
         excluded, reason = self.is_build_excluded(srpm_path)
```

Comment:
you should pass here path to SRPM instead of `built_srpm_url`

Reply:
Indeed.

Reply:
Passed `os.path.join(src_dir, os.path.basename(self.task.built_srpm_url))`.  

However, built_srpm_url doesn't seem to be set from anywhere.

Reply:
built_srpm_url comes from web-server: https://chat.almalinux.org/almalinux/pl/ttm61gngd38zmrie9kmx984zao